tests/validation/tests/v3_api/Jenkinsfile_deploy_and_test_support_matrix

```diff
@@ -138,6 +139,15 @@ node {
                 jobs = [:]
                 cluster_arr = env.RANCHER_CLUSTER_NAMES.split(",")
                 cluster_count = cluster_arr.size()
+                if (!RANCHER_UPGRADE_CHECK) {
```

Comment:
These should be env. RANCHER_UPGRADE_CHECK, env.RANCHER_VALIDATE_RESOURCES_PREFIX and env. RANCHER_CREATE_RESOURCES_PREFIX ?

Reply:
Do we need to check if (!RANCHER_UPGRADE_CHECK || env.RANCHER_UPGRADE_CHECK == "") ?